### ISSUE TITLE
Fork or PR?

### ISSUE THREAD

**Issue by hustf** [Khanzi/Spotify.jl#20]
Hello,
I'd like to say thank you for putting down so much work on a Julia implementation of this!
After using Spotify daily since 2009, I just wanted to make some more precise beat-per-minute playlists. I needed higher authorization than I found implemented here, and now this may not be a normal PR anymore I guess. Even though the web api is still incomplete, I'd like to get what's there from before working smoothly, and have tests working. If somebody can pick up the baton and keep improving from there, that would be good.

So basically, I'm curious about your plans for the repository? What kind of changes would you be willing to accept here?

**Comment by kwehmeyer** [comment]
Hey @hustf, the project was started by me and a friend who are both recent grads in specializing in data analytics and data science; needless to say we are not highly skilled programmers lol. Since starting the project our careers have gotten in the way and the project has stagnated. I'd be more than happy to give you full ownership on the condition that any code we wrote be credited in the future. Feel free to message me if you would like to chat about this more - Thanks for your interest!
